oracle/README.md

```diff
@@ -34,13 +37,36 @@ oracle/
 
 TODO: need UI for this to be relevant
 
+### Node
+In order to run the Oracle node make sure you are in the root of this project i.e `/path/to/oracle/<you are here>`
+
+Copy the `.env.example` file into a new file called `.env`.  Then insert your api key into the `API_URL`
+
+**_Note:_** You do not need an api key to run any tests
+
+
+Then start a local fuel-core instance
+```bash
+fuel-core run --chain packages/node/.chainConfig.json
+```
+
+Then deploy an instance of the Oracle contract
+```bash
+forc deploy --path packages/contract --url localhost:4000 --unsigned
+```
+
+Finally start the Oracle node
+```bash
+cargo run
+```
```

Comment:
```suggestion

The node can be started by executing the following steps:

1. Change into the `Oracle` directory

```bash
cd <path>/sway-applications/oracle/<you are here>
```

2. Copy and paste the `.env.example` file into a new file called `.env`

```bash
cp node/.env.example node/.env
```

3. In the newly copied `.env` file there is a variable `API_URL` which ends with `<your api key here>`. This section should be replaced with your API key.

4. Start `fuel-core`

```bash
fuel-core run --chain packages/node/.chainConfig.json
```

5. Deploy the Oracle contract

```bash
forc deploy --path packages/contract --url localhost:4000 --unsigned
```

6. Start the Oracle node

```bash
cargo run
```
```

This looks weird in the suggestion on github. I suggest checking this out locally to make it look good.

Points to note: 

1. This does not tell the reader anything about acquiring an API key. That is confusing. "Put this mysterious object that I've told you nothing about here". How would a user know what to do to acquire such an object?
2. Steps 4 and 5 seem odd to me. If I only want to "run a node" then why am I required to run some `fuel-core` and also deploy a contract? Moreover, `deploy` is a plugin which they must install separately so that might error for them.

The readme requires a bit more info

Reply:
Good suggestions I'll add more info/rework the readme